raspi-config

```diff
@@ -1582,13 +1582,13 @@ do_boot_rom() {
       else
         rpi-eeprom-update -d -f $FILNAME
         if [ "$INTERACTIVE" = True ]; then
-          whiptail --msgbox "Boot ROM reset to defaults" 20 60 2
+          whiptail --msgbox "Bootloader EEPROM config reset to defaults" 20 60 2
         fi
       fi
     else
       rpi-eeprom-update
       if [ "$INTERACTIVE" = True ]; then
-        whiptail --msgbox "Boot ROM not reset to defaults" 20 60 2
+        whiptail --msgbox "Boot EEPROM config left unchanged" 20 60 2
```

Comment:
Did you miss out a "Bootloader" here?

Reply:
I will fix it later today.

Reply:
> egg chasing on the telly

Today I learned a new phrase :rugby_football:

Reply:
> > egg chasing on the telly
> 
> Today I learned a new phrase 🏉

Every day's a school day 😀

Reply:
Fixed